src/background/Wallet/helpers/mask.ts

```diff
@@ -9,7 +9,10 @@ function maskMnemonic(
   return mnemonic ? { phrase: '<phrase>', path: mnemonic.path } : null;
 }
 
-export function maskWallet(wallet: ExternallyOwnedAccount | BareWallet) {
+/** TODO: rename to "removeSensitiveValues"? or something better? */
+export function maskWallet<T extends ExternallyOwnedAccount | BareWallet>(
```

Comment:
`maskSensitiveData` :)

Reply:
Well, none of these fit as this helper basically removes the value by replacing it with a fixed string